### PR TITLE
17946 Hide Alter Now button for BEN related alteration of numbered company

### PR DESCRIPTION
**Pull request by chenhongjing**
*Issue #:* /bcgov/entity#17946

*Description of changes:*
- Hid Alter Now button for BEN related numbered company
- Changed contact phone number for alteration
- app version=5.2.5

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-719-0nddoxpl.web.app

File: src/components/existing-request/nr-approved-gray-box.vue [hunk 1]
```diff
@@ -119,13 +119,13 @@
...
             </v-icon>&nbsp;Canada and U.S. Toll Free:
-            <a href="tel:+1877-370-1033">1-877-370-1033</a>
+            <a href="tel:+1877-526-1526">1-877-526-1526</a>
           </p>
           <p>
             <v-icon small>
               mdi-phone
             </v-icon>&nbsp;Victoria Office:
-            <a href="tel:250-370-1033">250-370-1033</a>
+            <a href="tel:250-387-7848">250-387-7848</a>
```
Comment (severinbeauvais) [inline src/components/existing-request/nr-approved-gray-box.vue:128]: This gray box, and the numbered company bullets below, aren't unique to alteration NRs. Please confirm with Mihai or Janis that it's OK to update the phone number for all NRs.

Comment (chenhongjing) [reply src/components/existing-request/nr-approved-gray-box.vue:128]: No, this is unique to alteration NRs.

Comment (severinbeauvais) [reply src/components/existing-request/nr-approved-gray-box.vue:128]: I'm sorry --  you are right about this component.

However, the code in numbered-company-bullets.vue appears to be for conversions only, so maybe you don't need to change the phone numbers there?

Comment (chenhongjing) [reply src/components/existing-request/nr-approved-gray-box.vue:128]: Conversion aka alteration 😸 so both need the change.

Comment (severinbeauvais) [reply src/components/existing-request/nr-approved-gray-box.vue:128]: Oh, I keep forgetting that. You're right again! Resolving.

**Review by severinbeauvais — APPROVED** [review]

**Review by JazzarKarim — APPROVED** [review]

**Pull request merged**